tests/builder.rs

```diff
@@ -1,4 +1,6 @@
 use incrementalmerkletree::{bridgetree::BridgeTree, Hashable, Tree};
```

Comment:
split in a different file for issuance tests

Reply:
done